package-lock.json

```diff
@@ -17988,11 +18038,21 @@
 				"lodash": "^4.17.21",
 				"memize": "^1.1.0",
 				"react-autosize-textarea": "^7.1.0",
-				"react-spring": "^8.0.19",
 				"redux-multi": "^0.1.12",
 				"rememo": "^3.0.0",
 				"tinycolor2": "^1.4.2",
 				"traverse": "^0.6.6"
+			},
+			"dependencies": {
```

Comment:
I'm not sure why it is here. It looks like it can be removed.

Reply:
Good catch, updated in https://github.com/WordPress/gutenberg/pull/30979/commits/317f1d87b38918e4fc60cf3d76fa7a3b4335b83c. 

The only dependencies for web should be the following:

<img width="760" alt="CleanShot 2021-09-13 at 09 28 20@2x" src="https://user-images.githubusercontent.com/1270189/133121755-564ee380-4578-4557-ae53-111f80d1d9b2.png">

Maybe it was a stray item from when I was testing other imports. To fix this, I removed the entry for `"@react-spring/web": "^9.2.4",` in package.json, `npm install`, then added it back with another `npm install` and it didn't appear to return.